### PR TITLE
initial setup changes & contact page namespace

### LINKED ISSUE #1068: contact page namespace

### PR DESCRIPTION
**Pull request by edwingl10**
## Motivation and Context

Fixes #1068 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] User visible change (users will notice UI or functional changes)

## How Has This Been Tested?

- [ ] Existing or new tests cover my changes.
- [ ] Manually tested locally or on the Render PR Server.

## Previewing Changes

1. Click the `onrender.com` URL the **render `bot`** posted below.
2. go to about page
3. change language

### CODE REVIEW

File: src/i18n/setup.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
-import TranslationsEn from './translations/english.json';
-import TranslationsSp from './translations/spanish.json';
+import contactEn from './translations/english/contactEn.json';
```
Comment (josueetcom) [inline src/i18n/setup.js:4]: Can we place these files under a structure like `locales/en/contact.json` instead? This seems to match up more closely with the extraction docs and some guides like this one:
https://medium.com/swlh/internationalization-with-react-i18next-360a452e8809

Comment (edwingl10) [reply src/i18n/setup.js:4]: yep, made the change. I made folders for english and spanish translations

**Comment by edwingl10** [comment]
Made some structure changes to closely match up the setup in https://medium.com/swlh/internationalization-with-react-i18next-360a452e8809.

I also commented out some tests that were failing. I will fix them in the other stacked PR's. If I fixed them in this PR, it would have probably required more files to be modified.

File: src/i18n.js [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
   .use(LanguageDetector)
   .init({
-    resources,
     supportedLngs: ['en', 'es'],
     fallbackLng: 'en',
+    backend: {
+      loadPath: `${process.env.PUBLIC_URL}/locales/{{lng}}/{{ns}}.json`,
+      addPath: `${process.env.PUBLIC_URL}/locales/add/{{lng}}/{{ns}}`,
+    },
+    ns: ['contact'],
```
Comment (josueetcom) [inline src/i18n.js:17]: If we merge this as is, we'll lose all the translations except for the ones added in the `contact` namespace. Can we instead in this PR rename the existing translation files as part of a general/common namespace and load that everywhere we need it?

**Comment by edwingl10** [comment]
Alright made more structure changes and small fixes. 
I mostly followed [this](https://www.i18next.com/how-to/add-or-load-translations)

File: public/locales/es/contact.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "titleTag": "Contacto",
+  "contactUs": "Contactanos",
```
Comment (josueetcom) [inline public/locales/es/contact.json:3]:
```suggestion
  "contactUs": "Contáctanos",
```

File: src/components/ScholarshipList.test.jsx [hunk 4]
```diff
@@ -44,8 +46,6 @@ beforeEach(() => {
 test('renders no results', async () => {
   renderWithProviders(<ScholarshipList />);
 
-  expect(screen.getByTestId('progress')).toBeInTheDocument();
```
Comment (josueetcom) [inline src/components/ScholarshipList.test.jsx:47]: Oh, why was this removed?

File: src/i18n.js [hunk 5]
```diff
@@ -1,19 +1,23 @@
...
-  en: { translation: TranslationsEn },
-  es: { translation: TranslationsSp },
-};
+import resourcesToBackend from 'i18next-resources-to-backend';
 
 i18n
   .use(initReactI18next)
   .use(LanguageDetector)
+  .use(
+    resourcesToBackend((lng, ns, clb) => {
```
Comment (josueetcom) [inline src/i18n.js:10]: I suggest using slightly more readable names:
```suggestion
    resourcesToBackend((lang, ns, callback) => {
```

File: src/index.jsx [hunk 6]
```diff
@@ -1,13 +1,18 @@
...
 import App from './App';
 import * as serviceWorker from './serviceWorker';
-import './i18n/setup';
+import { I18nextProvider } from 'react-i18next';
+import i18n from './i18n';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Suspense fallback="loading">
```
Comment (josueetcom) [inline src/index.jsx:11]: Is it possible to have no fallback or the empty string? I just tried simulating a slow 3G connection and it took like 20 seconds before this showed up and then another 5 seconds for the rest of the home page to load.

**Review by josueetcom — APPROVED** [review]

File: src/components/ScholarshipList.test.jsx (hunk 4 above)
Comment (edwingl10) [reply src/components/ScholarshipList.test.jsx:47]: It was causing the test to fail. Im guessing it was because Suspense took over and rendered "loading"

File: src/index.jsx (hunk 6 above)
Comment (edwingl10) [reply src/index.jsx:11]: without the `Suspense` I get 

"Error: Header suspended while rendering, but no fallback UI was specified.
Add a <Suspense fallback=...> component higher in the tree to provide a loading indicator or placeholder to display."

Comment (edwingl10) [reply src/index.jsx:11]: I changed it to an empty string instead
